types/ember-animated/animated-container.d.ts

```diff
@@ -1,6 +1,8 @@
 import Component from "@glint/environment-ember-loose/ember-component";
 
 interface AnimatedContainerSignature {
+  /** Multiple tags supported via `tag` arg, all should still be base HTMLElement. */
+  Element: HTMLElement;
```

Comment:
```suggestion
  Element: Element;
```

SVGElements aren't HTMLElements

Reply:
Alternatively, we could do something fancy where we take a generic for the `tag` and then use Glint's `ElementForTagName` here.

Reply:
That sounds very promising, didn't know that was a thing! I can look into that.

Reply:
Updated, that's a cool feature, I like it!

Reply:
@dwilhemli want to copy my tests from https://github.com/Gavant/glint-template-types/pull/28 ?

Reply:
I didn't expect you to get back to this so quickly!

Reply:
done, though I'm getting some type errors locally, unsure if it is just an environment issue or a real problem